bindings/cs/MongoDB.Libmongocrypt/Status.cs

```diff
@@ -32,12 +32,30 @@ public Status()
             _handle = Library.mongocrypt_status_new();
         }
 
+        public Status(StatusSafeHandle handle)
+        {
+            _handle = handle;
+        }
+
         public void Check(IStatus status)
         {
             status.Check(this);
```

Comment:
This is weird... calling the same method with the arguments reversed?

I can't tell where this new overload is defined.

Reply:
this is an old method :)

Reply:
I realize it's an old method. OK... I will ignore this weirdness.